risc0/zkvm/src/host/protos/api.proto

```diff
@@ -127,9 +127,9 @@ message IdentityP254Result {
 }
 
 message ExecutorEnv {
+  reserved 3;
   Asset binary = 1;
   map<string, string> env_vars = 2;
-  repeated string slice_ios = 3;
```

Comment:
I don't think we can actually do this since the idea is to have newer clients interact with older servers. I think we need to leave the protobuf alone and adjust the code that uses it.